### PR TITLE
Add Flask + SQLAlchemy test setup

### PR DESCRIPTION
**Pull request by unflxw**
Add a test setup that uses Flask, SQLAlchemy and SQLite.

This showcases the behaviour where query spans are duplicated between SQLAlchemy and SQLite.

Part of appsignal/appsignal-python#11

### CODE REVIEW

File: python/flask-sqlalchemy/commands/run [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+hatch run build:me
+
+cd /app
+
+echo "Installing dependencies"
+pip3 install -r requirements.txt
+pip3 install /integration/dist/* --force-reinstall
+
+echo "Starting app"
+#/commands/processmon processmon.toml
```
Comment (tombruijn) [inline python/flask-sqlalchemy/commands/run:20]: Processmon doesn't work for this type of app? I'm okay with not using it, but let's remove the comment then.

**Review by tombruijn — APPROVED** [review]
Bit of a hassle to figure out how to add the locally build agent again for Python, but it works!

image
image

File: python/flask-sqlalchemy/commands/run (hunk 1 above)
Comment (unflxw) [reply python/flask-sqlalchemy/commands/run:20]: I don't remember why I disabled processmon here (this is copy-pasted from the `flask` test setup, where it is also disabled) -- might be that Flask itself does auto-reload?

**Comment by unflxw** [comment]
> @tombruijn Bit of a hassle to figure out how to add the locally build agent again for Python, but it works!

Would you write it down here or in a relevant README for future reference? I have my "hacked" method of copy-pasting binaries around.

**Pull request merged**